children/tests/test_api.py

```diff
@@ -1063,6 +1078,134 @@ def test_get_past_events(
     )
 
 
+def test_get_past_events_including_external_ticket_system_events(
+    snapshot, guardian_api_client, child_with_user_guardian, project, venue
+):
+    variables = {"id": to_global_id("ChildNode", child_with_user_guardian.id)}
+    duration = 30
```

Comment:
`duration_mins` or `duration_minutes` would IMHO be clearer than `duration`. Event.duration is already named badly so it starts there. So, maybe that could be renamed there too although that's not necessary in this PR.

Reply:
Yeah agree. And whether it actually should be minutes at all is another good question 😄 However, changing just the field's name would require also changes to two UI's and therefore not worth it IMO.